rio_tiler/colormap.py

```diff
@@ -8,13 +8,18 @@
 import attr
 import numpy
 
-from .errors import (
+from rio_tiler.errors import (
```

Comment:
I've changed my mind about absolute/relative import. IMO it's much more cleaner with absolute import